.gitignore

```diff
@@ -45,4 +45,3 @@ build/
 **/*.cmd
 **/*.a
 *.elf
-src/utils/src/version_gen.c
```

Comment:
Why is the version file removed from .gitignore? It likely should not be committed.

Reply:
The path to the generated file is `build/src/utils/src/version_gen.c`, that is not in the `/src` dir which means that it is already ignored by the `build/` entry in `.gitignore`